staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go

```diff
@@ -198,6 +200,13 @@ func (o TopNodeOptions) RunTopNode() error {
 		} else {
 			availableResources[n.Name] = n.Status.Capacity
 		}
+
+		swapCapacity := int64(0)
+		if n.Status.NodeInfo.Swap != nil && n.Status.NodeInfo.Swap.Capacity != nil {
```

Comment:
We should check for NodeInfo also.

Reply:
I don't think we should. It's not a pointer, but rather a by-value field. Am I missing anything?

Reply:
Itamar is correct here.